### ISSUE TITLE
running ":skills" crashes CLI

### ISSUE THREAD

**Issue by fermulator** [MycroftAI/mycroft-core#2523]
* Linux Mint 18.X
```
mycroftai@mycroftai ~ $ cat /etc/issue
Linux Mint 18.2 Sonya \n \l
```
* Installed as per https://mycroft-ai.gitbook.io/docs/using-mycroft-ai/get-mycroft/linux#installing-via-git-clone
* mycroft latest as of a few days ago 
```
mycroftai@mycroftai ~/mycroft-core $ grep CORE_VERSION_ mycroft/version/__init__.py 
CORE_VERSION_MAJOR = 20
CORE_VERSION_MINOR = 2
CORE_VERSION_BUILD = 1
```

```
(.venv) mycroftai@mycroftai ~/mycroft-core $ which python
/home/mycroftai/mycroft-core/.venv/bin/python
(.venv) mycroftai@mycroftai ~/mycroft-core $ python --version
Python 3.5.2

(.venv) mycroftai@mycroftai ~/mycroft-core $ pip list
Package                       Version   
----------------------------- ----------
adapt-parser                  0.3.4     
alabaster                     0.7.12    
appdirs                       1.4.3     
astral                        1.4       
atomicwrites                  1.3.0     
attrs                         19.3.0    
Babel                         2.8.0     
beautifulsoup4                4.8.2     
casttube                      0.2.0     
certifi                       2019.11.28
cffi                          1.14.0    
chardet                       3.0.4     
click                         7.1.1     
colorama                      0.4.3     
convertdate                   2.2.0     
cov-core                      1.15.0    
coverage                      4.5.4     
coveralls                     1.8.2     
cryptography                  2.8       
ddg3                          0.6.6     
Deprecated                    1.2.7     
docopt                        0.6.2     
docutils                      0.16      
entrypoints                   0.3       
fann2                         1.0.7     
fasteners                     0.14.1    
feedparser                    5.2.1     
flake8                        3.7.9     
fuzzywuzzy                    0.14.0    
gitdb                         4.0.2     
GitPython                     3.1.0     
google-api-python-client      1.6.4     
gTTS                          2.0.4     
gTTS-token                    1.1.3     
holidays                      0.10.1    
httplib2                      0.17.0    
humanhash3                    0.0.6     
idna                          2.7       
ifaddr                        0.1.6     
imagesize                     1.2.0     
importlib-metadata            1.6.0     
inflect                       3.0.2     
inflection                    0.3.1     
jaraco.itertools              4.4.2     
Jinja2                        2.11.1    
lazy                          1.4       
lingua-franca                 0.2.0     
MarkupSafe                    1.1.1     
mccabe                        0.6.1     
monotonic                     1.5       
more-itertools                8.2.0     
msk                           0.3.14    
msm                           0.8.7     
mtranslate                    1.6       
multi-key-dict                2.0.3     
num2words                     0.5.10    
oauth2client                  4.1.3     
packaging                     20.3      
padaos                        0.1.9     
padatious                     0.4.6     
pako                          0.2.3     
pathlib2                      2.3.5     
petact                        0.1.2     
Pillow                        6.2.1     
pip                           20.0.2    
pluggy                        0.13.1    
pocketsphinx                  0.1.0     
precise-runner                0.2.1     
protobuf                      3.11.3    
psutil                        5.2.1     
py                            1.8.1     
pyalsaaudio                   0.8.2     
pyasn1                        0.4.8     
pyasn1-modules                0.2.8     
PyAudio                       0.2.11    
PyChromecast                  3.2.2     
pycodestyle                   2.5.0     
pycparser                     2.20      
pyee                          5.0.0     
pyflakes                      2.1.1     
PyGithub                      1.47      
Pygments                      2.6.1     
pyjokes                       0.5.0     
PyJWT                         1.7.1     
PyMeeus                       0.3.7     
pyOpenSSL                     19.1.0    
pyowm                         2.6.1     
pyparsing                     2.4.6     
pyserial                      3.0       
pytest                        5.2.4     
pytest-cov                    2.8.1     
python-dateutil               2.6.0     
python-Levenshtein            0.12.0    
python-vlc                    1.1.2     
pytz                          2017.2    
PyYAML                        5.1.2     
requests                      2.20.0    
requests-futures              0.9.5     
responses                     0.10.12   
rsa                           4.0       
setuptools                    46.1.3    
six                           1.13.0    
smmap                         3.0.1     
snowballstemmer               2.0.0     
soupsieve                     2.0       
SpeechRecognition             3.8.1     
Sphinx                        2.2.1     
sphinx-rtd-theme              0.4.3     
sphinxcontrib-applehelp       1.0.2     
sphinxcontrib-devhelp         1.0.2     
sphinxcontrib-htmlhelp        1.0.3     
sphinxcontrib-jsmath          1.0.1     
sphinxcontrib-qthelp          1.0.3     
sphinxcontrib-serializinghtml 1.1.4     
tornado                       6.0.3     
typing                        3.7.4.1   
tzlocal                       1.3       
uritemplate                   3.0.1     
urllib3                       1.24.3    
uuid                          1.30      
wcwidth                       0.1.9     
websocket-client              0.54.0    
wheel                         0.34.2    
wikipedia                     1.4.0     
wolframalpha                  3.0       
wrapt                         1.12.1    
xmlrunner                     1.7.7     
xmltodict                     0.12.0    
xxhash                        1.4.3     
zeroconf                      0.24.5    
zipp                          1.2.0     

```

## Try to provide steps that we can use to replicate the Issue

For example: 

1. This is a fresh Linux installation via `dev_setup`
2. Running `:skills` per the help in CLI mode crashes

## Be as specific as possible about the expected condition, and the deviation from expected condition. 

```
mycroftai@mycroftai ~/mycroft-core $ ./start-mycroft.sh cli

Already up-to-date.
Initializing...
Starting cli
2020-03-31 21:58:41.540 | INFO     |  6577 | mycroft.messagebus.load_config:load_message_bus_config:33 | Loading message bus configs

```

one can type `:help` to see commands, from there I saw `:skills` (which should list skills)
instead it crashes
```
Traceback (most recent call last):
  File "/usr/lib/python3.5/runpy.py", line 184, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.5/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/mycroftai/mycroft-core/mycroft/client/text/__main__.py", line 71, in <module>
    main()
  File "/home/mycroftai/mycroft-core/mycroft/client/text/__main__.py", line 65, in main
    curses.wrapper(gui_main)
  File "/usr/lib/python3.5/curses/__init__.py", line 94, in wrapper
    return func(stdscr, *args, **kwds)
  File "/home/mycroftai/mycroft-core/mycroft/client/text/text_client.py", line 1316, in gui_main
    if handle_cmd(line[1:]) == 1:
  File "/home/mycroftai/mycroft-core/mycroft/client/text/text_client.py", line 1145, in handle_cmd
    scr.get_wch()  # blocks
_curses.error: no input
```

given the stack trace
```
mycroftai@mycroftai ~ $ dpkg --list | grep ncurses
ii  libncurses5:amd64                           6.0+20160213-1ubuntu1                        amd64        shared libraries for terminal handling
ii  libncursesw5:amd64                          6.0+20160213-1ubuntu1                        amd64        shared libraries for terminal handling (wide character support)
ii  mtr-tiny                                    0.86-1ubuntu0.1                              amd64        Full screen ncurses traceroute tool
ii  ncurses-base                                6.0+20160213-1ubuntu1                        all          basic terminal type definitions
ii  ncurses-bin                                 6.0+20160213-1ubuntu1                        amd64        terminal-related programs and man pages
```

## Provide log files or other output to help us see the error

History ===============================================================================    Log Output Legend ======================================================= Mic Level ===
 Send me debug info                                                                        DEBUG output
 >> I don't understand.                                                                    skills.log, other
 You're not working!                                                                       voice.log                                                                   --- 446.38
 >> I'm not sure I understood you.

kk....

**Comment by krisgesling** [comment]
Hey there, what is this running on is it a Linux install, a VM, Docker, custom Pi install or something else?

**Comment by forslund** [comment]
Also ncurses verion on the system could be of interest:

On an apt based system run

`apt list | grep ncurses`

and check which packages are marked as installed

**Comment by fermulator** [comment]
Sorry about that forgot platform information (updated main)
Will get `ncurses` shortly

**Comment by forslund** [comment]
Hmm, my ncurses are slightly newer...

```
dpkg --list | grep ncurses
ii  libncurses5:amd64                     6.0+20161126-1+deb9u2                       amd64        shared libraries for terminal handling
ii  libncurses5-dev:amd64                 6.0+20161126-1+deb9u2                       amd64        developer's libraries for ncurses
ii  libncursesw5:amd64                    6.0+20161126-1+deb9u2                       amd64        shared libraries for terminal handling (wide character support)
ii  libncursesw5-dev:amd64                6.0+20161126-1+deb9u2                       amd64        developer's libraries for ncursesw
ii  ncurses-base                          6.0+20161126-1+deb9u1                       all          basic terminal type definitions
ii  ncurses-bin                           6.0+20161126-1+deb9u1                       amd64        terminal-related programs and man pages
ii  ncurses-term                          6.0+20161126-1+deb9u1                       all          additional terminal type definitions
```

**Comment by forslund** [comment]
But I don't imagine that's the cause of this... :/

Will do some googling

**Comment by forslund** [comment]
Are you running the cli in text mode or in a desktop terminal emulator?

**Comment by fermulator** [comment]
not precisely sure what you mean, but I'm running it in whatever the default mint terminal is; all defaults/vanilla configs

**Comment by forslund** [comment]
Ok good info. I can't reproduce here so I might need to install Mint in VM and see...

**Comment by fermulator** [comment]
Confirmed even after switching to `dev` branch (from `master`) that this issue is still present.

**Comment by forslund** [comment]
I tried this on Linux Mint 19.3 and I couldn't reproduce the issue, going to see if I can get the exact version you're using.

**Comment by forslund** [comment]
Question: which version of Mint are you running? Cinnamon, KDE, Mate, Xfce?

**Comment by fermulator** [comment]
(see at the top? mint 18.3)  default window manager; it was a brand new vanilla installation purely for testing mycroft on Linux

**Comment by forslund** [comment]
Ok I'm guessing the default is Cinnamon then.

**Comment by forslund** [comment]
Finally got round to setting up Mint 18.2, and I can reproduce the issue. Great! Will keep you posted

**Comment by forslund** [comment]
Found a workaround for the issue. PR is up and waiting for review

**Comment by forslund** [comment]
This should be resolved now through #2591.

**Comment by forslund** [comment]
Closing this now, please reopen if the problem persist.
